src/test/java/com/teamscale/upload/NativeImageIT.java

```diff
@@ -362,8 +436,12 @@ private Arguments withInput(String input) {
 		}
 
 		private String[] toStringArray() {
-			List<String> arguments = new ArrayList<>(Arrays.asList("-s", url, "-u", user, "-a", accessKey, "-f", format,
-					"-p", project, "-t", partition));
+			List<String> arguments = new ArrayList<>(
+					Arrays.asList("-s", url, "-u", user, "-f", format, "-p", project, "-t", partition));
+			if (accessKey != null) {
+				arguments.add("-a");
+				arguments.add(accessKey);
+			}
```

Comment:
I would tend to tolerate the long-method finding here. I do not see a way to improve this by extracting methods...